INN/umbrella-currentorg
# Updated tags field to `select2` on error refresh

## Changes

This pull request makes the following changes:

- Added function to convert the tags field into a select2 field when the submit listing page is refreshed due to an error.

## Why



For #93 / a bug where the tags field was showing up in its default state if the form was submitted with an error.

## Testing/Questions

Features that this PR affects:

- Submit listing form functionality

Steps to test this PR:

1. Fill out listing form but leave out a required field
2. Submit listing and get taken to "error" page
3. Verify the tags field is still the select2 dropdown
